IDesign/IDesign.CommonResources/ResourceUtils.cs

```diff
@@ -51,17 +58,35 @@ public static string ResourceMessageToString(IResourceMessage ResMessage)
         public static string ResultToString(ICheckResult result)
         {
             var res = "";
-            if (result.GetElement() != null)
+            var messageResource = result.GetFeedback();
+
+            if (result.GetElement() != null && !ContainsCurlyBracketsWithDigits(messageResource))
             {
                 res += result.GetElement() + " | ";
             }
 
-            if (result.GetFeedback() != null)
+            if (messageResource != null)
             {
-                res += ResourceMessageToString(result.GetFeedback());
+                res += ResourceMessageToString(messageResource, result);
             }
 
             return res;
         }
+
+        private static bool ContainsCurlyBracketsWithDigits(IResourceMessage resMessage)
+        {
+            if (resMessage != null)
+            {
+                var resMessageString = GetResourceFromString(resMessage?.GetKey());
+                var regex = new Regex(@"\{\d\}");
```

Comment:
This will not match `{10}` maybe an extreme case but hard to debug when in production. correct regex: `\{\d+}\`